### PR TITLE
my Account landing page changes

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/js/button.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ $(function() {
                 $('.signin').css('display','none');
                 $('.signup').css('display','none');
                 $('.logout').css('display','inline-block');
+				$('.myaccount').css('display','inline-block');
```
Comment (rakeshkumar-verma) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/js/button.js:8]: @DeepaliAgrwl  My Account Button should be hidden for anonymous user.

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/_cq_dialog/.content.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
```
Comment (rakeshkumar-verma) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/_cq_dialog/.content.xml:2]: @DeepaliAgrwl  inner border title should be aligned as per design.

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/js/button.js (hunk 1 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/js/button.js:8]: this is done

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/_cq_dialog/.content.xml (hunk 2 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/_cq_dialog/.content.xml:2]: Done

**Pull request merged**